projects/palindrome/_posts/2020-10-07-kotlin.md

```diff
@@ -0,0 +1,81 @@
+---
+title: Palindrome in Kotlin
+layout: default
+last-modified: 2020-10-07
+featured-image:
+tags: [palindrome]
```

Comment:
```suggestion
tags: [palindromic-numbers]
```

Reply:
I'm changing the output to true or false.